### PR TITLE
Rewrite the exports with aliases in the declaration files as relative imports

### PR DESCRIPTION
**Pull request by cdagli**

### CODE REVIEW

File: dist/src/parser-engine.js [hunk 1]
```diff
@@ -97,11 +99,17 @@ class ParserEngine {
...
-            let filename = fileList[i];
-            this.processFile(filename);
+        let JSfileList = new Array();
+        this.walkSync(this.appRoot, JSfileList, ".js");
+        for (var i = 0; i < JSfileList.length; i++) {
+            let filename = JSfileList[i];
+            this.processJSFile(filename);
+        }
+        let TSFileList = new Array();
+        this.walkSync(this.appRoot, TSFileList, ".ts");
```
Comment (rifler) [inline dist/src/parser-engine.js]: how about `tsx` ?

**Comment by bgedik** [comment]
@duffman Do you think this could be merged, it will be very helpful for some of us.

File: dist/src/parser-engine.js (hunk 1 above)
Comment (cdagli) [reply dist/src/parser-engine.js]: We can add `tsx`, too. Are there any difference with ```ts``` import / export statements, do you have a sample?

**Comment by cdagli** [comment]
@duffman Any comments on that?

**Comment by ifiokjr** [comment]
Just checking, is this being merged soon, or am I better off creating a fork for the functionality?

**Comment by duffman** [comment]
I will have a look at this during the weekend, right now I´ve just seen
your comments, I have not fully understood what it is that possibly
would hinder the use of TSPath in your project, but as I said I will have a
look at it today or tomorrow...

There is a new 2.x release in the workshop right now that will change the
inner structure of the entire project so I´ll see if this is something
that is already fixed, in that case, a quick merge would be fine I guess!

Anyhow thanks for your involvement!

//Duffman

Den fre 7 dec. 2018 kl 07:45 skrev Ifiok Jr. <notifications@github.com>:

**Pull request closed without merging**